src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,135 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private final int maxSize = 10;
+    private int size = 0;
+    private Object[] array;
+
+    public ArrayList() {
+        array = new Object[maxSize];
+    }
+
+    private Object[] resize(int newLength) {
```

Comment:
```suggestion
    private Object[] resize() {
```
ти завжди однакові вхідні параметри передаєш, можна просто newLength всередині метода порахувати так як ти маєш доступ до необхідних змінних класу

Reply:
Спасибо за ревью!)